operations/iscsi_sbps/iscsi_sbps.md

```diff
@@ -3,27 +3,25 @@
 * [Introduction](#introduction)
 * [iSCSI SBPS solution details](#iscsi-sbps-solution-details)
 * [Steps to achieve SBPS](#steps-to-achieve-sbps)
-* [Steps to continue using DVS based projection](#steps-to-continue-using-dvs-based-projection)
-* [Steps to disable DVS and CPS](#steps-to-disable-dvs-and-cps)
 * [Glossary](#glossary)
 
 ## Introduction
 
 iSCSI based boot content projection solution named Scalable Boot Content Projection Service (SBPS)
-is an alternate boot content projection solution and planning to replace current Cray
-[Data Virtualization Service (DVS)](../../glossary.md#data-virtualization-service-dvs) to project boot content like `rootfs`
-and [Cray Programming Environment (CPE)](../../glossary.md#cray-programming-environment-cpe) images. SBPS is aimed to offer
-better reliability, availability, security, ease and speed of deployment and ease of management than DVS.
-The SBPS solution is offered from CSM 1.6 onwards and this solution is spread across different components, including:
+is a boot content projection solution that replaces the former Cray
+[Content Projection Service (CPS)](../../glossary.md#content-projection-service-cps) and
+[Data Virtualization Service (DVS)](../../glossary.md#data-virtualization-service-dvs).
+SBPS projects boot content like `rootfs` and [Cray Programming Environment (CPE)](../../glossary.md#cray-programming-environment-cpe) images.
+SBPS is aimed to offer better reliability, availability, security, ease and speed of deployment and ease of management than CPS/DVS.
+SBPS was introduced in CSM 1.6. CPS and DVS are removed in CSM 1.7.
+
+The SBPS solution is spread across different components, including:
 
 * [Boot Orchestration Service (BOS)](../../glossary.md#boot-orchestration-service-bos)
 * [Cray Operating System (COS)](../../glossary.md#cray-operating-system-cos)
```

Comment:
FWIW I believe the remaining packages in COS/USS that helped support SBPS will be present in CSM 1.7.  We will them remove these components from USS in the following release in 2026.  Given that, it seems like this line could be removed.